terraform_utils/hcl.go

```diff
@@ -187,12 +190,8 @@ func terraform12Adjustments(formatted []byte, mapsObjects map[string]struct{}) [
 
 // Sanitize name for terraform style
 func TfSanitize(name string) string {
-	name = strings.Replace(name, " ", "", -1)
-	//name = strings.Replace(name, "-", "_", -1)
-	name = strings.Replace(name, "*", "--", -1)
-	name = strings.Replace(name, ".", "--", -1)
-	name = strings.Replace(name, ":", "--", -1)
-	name = strings.Replace(name, "/", "--", -1)
+	name = unsafeChars.ReplaceAllString(name, "--")
+	name = "tfer--" + name
```

Comment:
Why we need "tfer--" prefix?

Reply:
This suffix is necessary because we can have a resource on AWS that starts with a number, example: `1234`. Although numbers are allowed on Terraform identifiers, these identifiers should start a letter, just like a variable on Go.

I had this problem when trying to parse some Security Groups from AWS, there are tools that automatically create security groups on AWS starting with numbers.

Reply:
I’ve explained why just not to remove these numbers from the beginning of the string here: https://github.com/GoogleCloudPlatform/terraformer/pull/220#issuecomment-540014303

Another thing that could happen is: a resource name that just have numbers would generate an employ string as Terraform identifier if we use New Relic’s provider approach.